app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt

```diff
@@ -115,7 +117,8 @@ fun ParticipantTile(
                         bottom.linkTo(parent.bottom)
                         start.linkTo(parent.start)
                         end.linkTo((parent.end))
-                    },
+                    }
+                    .widthIn(max = dimensions().onGoingCallTileUsernameMaxWidth),
```

Comment:
Is this `widthIn` necessary for it?

Can't we instead put the Text to fill the parent only?

I'm not sure if hardcoding these dimensions will resize well on all screen sizes.